docs/developer-docs/functionality/bitcoin/bitcoin.md

```diff
@@ -0,0 +1,219 @@
+    > A get_utxos_request without the optional filter results in a request that considers the full blockchain, which is equivalent to setting min_confirmations to 0.
+
+-   bitcoin_get_balance: Given a get_balance_request, which must specify a Bitcoin address and a Bitcoin network (mainnet or testnet), the function returns the current balance of this address in Satoshi (10\^8 Satoshi = 1 Bitcoin) in the specified Bitcoin network. The same address formats as for bitcoin_get_utxos are supported.
+
+-   bitcoin_send_transaction: Given a send_transaction_request, which must specify a blob of a Bitcoin transaction and a Bitcoin network (mainnet or testnet), several checks are performed, and, if successful, the transaction is forwarded to the Bitcoin network.
+
+-   bitcoin_get_current_fee_percentiles: The transaction fees in the Bitcoin network change dynamically based on the number of pending transactions. It must be possible for a canister to determine an adequate fee when creating a Bitcoin transaction.\
+    > This function returns the 100 fee percentiles, measured in millisatoshi/byte (10\^3 millisatoshi = 1 satoshi), over the last 10,000 transactions, i.e., over the transactions in the last approximately 4-10 blocks.
+
+We refer to the \[Internet Computer Interface Specification\](https://internetcomputer.org/docs/current/references/ic-interface-spec) for the details of the Bitcoin integration API.
+
+## Development, Pre-Production, and Production Stage
+
+The Bitcoin functionality including threshold ECDSA is available in all stages required for the development life cycle on the IC:
+
+-   The *SDK* for local development of canisters;
+
+-   IC mainnet support as the pre-production environment for final testing on *Bitcoin testnet*;
+
+-   IC mainnet support as the production environment for the release using *Bitcoin mainnet*.
+
+### Local SDK
+
+In the typical canister development workflow, canisters on the IC are compiled and run in the local environment using the Motoko or Rust canister SDK during their development. Thus, the SDK is the first stage of the development workflow. The SDK has been enabled to support both the Bitcoin integration and threshold ECDSA management canister APIs.
+
+In contrast to the mainnet deployments of the feature, which integrate with Bitcoin testnet and Bitcoin mainnet, respectively, the SDK integrates with a locally-running bitcoind node in regression testing (regtest) mode. Using bitcoind in regtest mode is the preferred way of Bitcoin development. To facilitate our developers as best as possible, we integrated the SDK with bitcoind in regtest mode to bring the best Bitcoin development experience to the IC. Both development and automated testing of smart contracts are first done in the local environment.
+
+The Bitcoin adapter of the single replica running the local SDK environment connects to the local bitcoind node instead of multiple nodes of Bitcoin testnet or mainnet.
```

Comment:
```suggestion
The Bitcoin adapter of the single replica running the local SDK environment connects to the local bitcoind node instead of multiple nodes of Bitcoin testnet or mainnet. To see the relevant flags on dfx, please look at the output of `dfx start --help`.
```